packages/block-editor/src/components/link-control/link-preview.js

```diff
@@ -96,7 +96,8 @@ export default function LinkPreview( {
 
 	return (
 		<div
-			aria-label={ __( 'Currently selected' ) }
+			role="group"
+			aria-label={ __( 'Manage link' ) }
```

Comment:
The `Currently selected` is quite old code now and I'm really pleased to get input on improving this 👍